app/gui/view/component-browser/component-group-view/src/lib.rs

```diff
@@ -0,0 +1,224 @@
+//! An EnsoGL implementation of a Component Group View.
+//!
+//! A Component Group View is displayed as a header (containing the name of a group of Enso
```

Comment:
Why this thing is called "Component Group View" instead of just "Component Group"? I know that we have "List View", but this is because "List" is a data structure. Other components, like "Text Area" are not named "Text Area View". @farmaazon, what do you think of it?